roles/openshift_repos/meta/main.yml

```diff
@@ -11,4 +11,5 @@ galaxy_info:
     - 7
   categories:
   - cloud
-dependencies: []
+dependencies:
+- role: openshift_inventory
```

Comment:
I don't see a way to add this globally via ansible.cfg or something. I agree it would seem cleaner but on the other hand it introduces spooky action at a distance, so maybe it's best that it doesn't seem possible.